css/style.css

```diff
@@ -0,0 +1,10 @@
+#image{
+    width: auto;
+    max-width: 100%;
+    display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+#title{
+    text-align: center;
+}
```

Comment:
Great use of CSS selectors! Nevertheless, remember that the Linters file can become pretty useful here, you can add it and it will tell you how to improve your style. I recommend you to use this link to set them up https://github.com/microverseinc/linters-config/tree/master/html-css

Reply:
Splendid work 🙌🔥